labs/RivePlayer/src/AnimationTimer.cs

```diff
@@ -52,7 +52,8 @@ public void Stop()
 
     private async void InvalLoopAsync(int continuationToken)
     {
-        while (continuationToken == _invalLoopContinuationToken)
+        while (continuationToken == _invalLoopContinuationToken &&
+               rivePlayerRef.TryGetTarget(out var rivePlayer))
```

Comment:
This is still going to just check for the reference once and then hold it in the local loop which will keep the control loaded. @Sergio0694 suggestions on how to break the handle cycle here with the render loop?

Reply:
Ah, was just thinking we could split this into two parts (the loop and the action):

```cs
    private async void InvalLoopAsync(int continuationToken)
    {
        while (continuationToken == _invalLoopContinuationToken)
        {
             PerformLoopAction();
             await Task.Delay(timeSpanPerFrame);
        }
    }
    
    private void PerformLoopAction()
    {
        if (rivePlayerRef.TryGetTarget(out var rivePlayer))
        {
            if (!rivePlayer.IsLoaded)
            {
                Stop():
                return;
            }
            
            rivePlayer.InvalidateAnimation();
        }
        else
        {
            Stop();
        }
    }
```